src/index.ts

```diff
@@ -0,0 +1,125 @@
+import { SecretManagerServiceClient, protos } from "@google-cloud/secret-manager";
+
+/**
+ * Filter function to filter secrets by name that dont have to be loaded
+ */
+type FilterFunction = (name: string) => boolean;
```

Comment:
@neotob 

I suggest to also pass the secret object itself. 

```suggestion
/**
 * Filter function to filter secrets by name that dont have to be loaded
 */
type FilterFunction = (params: {
  /**
   * The name of the secret (without the path)
   */
  name: string;

  /**
   * The secret itself
   */
  secret: protos.google.cloud.secretmanager.v1.ISecret;
}) => boolean;
```